drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -1119,3 +1120,61 @@ def workbench_scenarios():
                 "<vertical_demo><drag-and-drop-v2 mode='assessment' max_attempts='3'/></vertical_demo>"
             ),
         ]
+
+    def _strip_html_tags_and_extra_whitespaces(self, raw_html):
+        cleanr = re.compile('<.*?>')
+        cleantext = str(re.sub(cleanr, '', raw_html))
+        cleantext = " ".join(re.split(r"\s+", cleantext, flags=re.UNICODE)).strip()
+        return cleantext
```

Comment:
Instead of this, please use `bleach`, then you can simply call `bleach.clean(raw_html)`. 
You will also need to add it to the requirements file and setup.py. 

It's already installed with edx-platform so it's OK to have this as a dependency.

Reply:
Thanks :+1: